api/builder.go

```diff
@@ -6,103 +6,51 @@ import (
 	"mergemock/rpc"
 
 	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
-	"github.com/ethereum/go-ethereum/core/types"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
 	gethRpc "github.com/ethereum/go-ethereum/rpc"
 	"github.com/ethereum/go-ethereum/trie"
-	"github.com/holiman/uint256"
+	"github.com/flashbots/mev-boost/types"
```

Comment:
Please remove this dependency. We should redefine all the types in `mergemock`. This let's us release new versions without waiting on upstream components.